executor/adapter.go

```diff
@@ -1193,6 +1194,7 @@ func (a *ExecStmt) SummaryStmt(succ bool) {
 	if tikvExecDetailRaw != nil {
 		tikvExecDetail = *(tikvExecDetailRaw.(*util.ExecDetails))
 	}
+	execDetail.BackoffSleep["waitLockLease"] = sessVars.WaitLockLeaseTime
```

Comment:
The name `waitLockLease` is a little too general. Some name can reflect it is a lock for cache table?